src/Sentry.Unity.Editor/Sentry.Unity.Editor.csproj

```diff
@@ -3,11 +3,14 @@
     <!-- This assembly only stays in the Editor, so embed pdbs in the dll to help debugging -->
     <DebugType>embedded</DebugType>
     <OutDir>$(PackageEditorPath)</OutDir>
+    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
   </PropertyGroup>
   <ItemGroup>
-    <ProjectReference Include="../sentry-dotnet/src/Sentry/Sentry.csproj" Private="false" />
-    <ProjectReference Include="../Sentry.Unity/Sentry.Unity.csproj" Private="false" />
+    <ProjectReference Include="../sentry-dotnet/src/Sentry/Sentry.csproj" Private="false" IncludeAssets="compile" />
```

Comment:
Why is `IncludeAssets="compile"` added?  I'm also surprised you need `Private="false"`.

Reply:
Without those all the dependencies get copied to the output directory as well, effectively duplicating them within the package.